types/luxon/src/datetime.d.ts

```diff
@@ -642,7 +708,7 @@ export class DateTime {
      * @example
      * DateTime.local(2017, 5, 25).ordinal //=> 145
      */
-    get ordinal(): number | DateTime;
+    get ordinal(): number;
```

Comment:
this one was ok, the change is breaking I believe, `number` for NaN, otherwise date time like object:
https://github.com/moment/luxon/blob/0cbffd9c87eca6ac9a75dce124ce890dd945bc78/src/impl/conversions.js#L88

Reply:
The get ordinal() always returns the ordinal property of that object, which is always a number
https://github.com/moment/luxon/blob/d95ddf12b3d9b116054a8c715c4aa596c4a34454/src/datetime.js#L1084
https://github.com/moment/luxon/blob/d95ddf12b3d9b116054a8c715c4aa596c4a34454/src/impl/conversions.js#L28

Reply:
yes, my bad, sorry!